src/esp/sensor/CameraSensor.cpp

```diff
@@ -28,6 +29,10 @@ CameraSensor::CameraSensor(scene::SceneNode& cameraNode,
   renderCamera_->setViewport(this->framebufferSize());
 }  // ctor
 
+CameraSensor::~CameraSensor() {
+  renderCamera_ = nullptr;
```

Comment:
Why?

Reply:
Hmm, was this one also a false positive?  Was to fix: https://clang.llvm.org/docs/analyzer/checkers.html#cplusplus-newdeleteleaks-c

It was orignally complaining about new RenderCamera() not having a corresponding deletion, but calling delete() on RenderCamera was causing tests to fail here so I tried removing the reference from it like we do in Simulator.cpp . Is there a better way to solve this issue? I am not that familiar with C++ memory management.

Reply:
Assigning a null to a raw pointer in a destructor doesn't achieve anything :) The LGTM "leak" report was a false positive, so no action is needed.